### PR TITLE
Medical Techfab + Bed Deconstruction Fix

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Noticed that Delta gave med a protolathe, and we now have enough items to justify a medical techfab. The only change to other lathes was removing health analyzers from the autolathe. All of the new lathe recipes are unlocked by biological technology.

**Changelog**

:cl: Rane
- add: Medbay now has their own techfab, pending mapping.
- tweak: Printing latex and nitrile gloves, sterile masks, mouth swabs, and health analyzers now requires a Medical Techfab and Biological Technology unlocked.
- fix: Beds now give back the cloth/durathread too when you deconstruct them.

### CODE REVIEW

**Comment by juliangiebel** [comment]
Making the blue part a new layer on top of the sec-fab icon would have been better.
You can just add new layers for different departments in the future and it saves on space.
icon

**Review by juliangiebel — APPROVED** [review]

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -302,6 +301,64 @@
       - CableHVStack
       - TimerTrigger
 
+- type: entity
+  parent: Protolathe
+  id: MedicalTechFab
+  name: Medical TechFab
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:307]:
```suggestion
  name: medical techfab
```

Comment (mirrorcult) [reply Resources/Prototypes/Entities/Structures/Machines/lathe.yml:307]: noticing that a lot of these are uppercased actually so if you wanna go through and change those you can

**Review by mirrorcult — APPROVED** [review]

**Comment by Elijahrane** [comment]
Made them all lowercase

**Pull request merged**